.travis.yml

```diff
@@ -58,6 +59,14 @@ env:
 install: .travis/install.sh functional-tests docs
 script:
 - export PYTHONPATH="${TRAVIS_BUILD_DIR}/.travis"
+# When we run cylc commands, there are processes being forked, that get a
+# new working directory. As .coveragerc contains relatives paths, it fails
+# to produce the correct coverage, unless we use absolute paths. The `sed`
+# call below tries to define the data_file, and sources locations for Travis.
+- sed -e "s|data_file=.coverage|data_file=${TRAVIS_BUILD_DIR}/.coverage|g; s|./bin|${TRAVIS_BUILD_DIR}/bin|g; s|./lib|${TRAVIS_BUILD_DIR}/lib|g" .coveragerc > /tmp/.coveragerc
+# And some tests fail if we touch files in the git working directory, due
+# to Cylc's version appearing with the "dirty" suffix. To avoid this, we
+# are using a new coveragerc created under the temporary directory.
 - coverage run .travis/cover.py
```

Comment:
I'm a bit confused as to why this change is showing up in your PR, when it already exists in master?!

Reply:
Bruno pointed out my version of .travis.yml was outdated so I just replaced my .travis.yml content with the master version...